change/@azure-msal-node-8b9bf4a1-ea08-4013-8a4d-a6deb79b04ce.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think this needs to be a major change type since it breaks v1 behavior

Reply:
This is going to v3 right?

Reply:
@sameerag yep